testing/testcafe/runner.js

```diff
@@ -115,3 +132,24 @@ function getArgs() {
         }
     });
 }
+
+function clearTestPage() {
```

Comment:
Let's move this function before ```createTestCafe```

Because this function is used before it's declaration :)

Reply:
There are other functions declared the same way - setTestingPlatform(), expandBrowserAlias()... I think that is not a big problem. This function will be move to separate js file soon. So, let's wait future changes. I am going to request your review for this change.